spec/config/mongoid_with_schema_map_uuid.yml

```diff
@@ -0,0 +1,27 @@
+test:
+  clients:
+    default:
+      database: mongoid_test
+      hosts:
+        <% SpecConfig.instance.addresses.each do |address| %>
+          - <%= address %>
+        <% end %>
+      options:
+        auto_encryption_options:
+          key_vault_namespace: 'admin.datakeys'
+          kms_providers:
+            local:
+              key: "z7iYiYKLuYymEWtk4kfny1ESBwwFdA58qMqff96A8ghiOcIK75lJGPUIocku8LOFjQuEgeIP4xlln3s7r93FV9J5sAE7zg8U"
+          schema_map:
+            blog_development.comments:
+              properties:
+                message:
+                  encrypt:
+                    keyId:
+                      - !ruby/object:BSON::Binary
+                        data: !binary |-
+                          R/AgNcxASFiiJWKXqWGo5w==
```

Comment:
Where is this generated from? Any reason we can't store UUIDs in a human-readable form in the config?

Reply:
This particular value came from https://github.com/mongodb/mongo-ruby-driver/pull/2429. To learn how FLE works in driver, please refer to https://www.mongodb.com/docs/ruby-driver/master/reference/client-side-encryption/.